src/middleware/jira-admin-permission-middleware.ts

```diff
@@ -0,0 +1,48 @@
+import { NextFunction, Request, Response } from "express";
+import { Installation } from "models/installation";
+import { JiraClient } from "models/jira-client";
+import { booleanFlag, BooleanFlags } from "config/feature-flags";
+
+export const setJiraAdminPrivileges = async (req: Request, claims: Record<any, any>, installation: Installation) => {
```

Comment:
nit: can we add a return type here?

Reply:
yes, yes we can. good catch!

Reply:
might almost be time to enforce this eslint rule with an error
`@typescript-eslint/explicit-function-return-type`